packages/components/psammead-caption/src/index.stories.jsx

```diff
@@ -39,4 +39,13 @@ storiesOf('Caption', module)
       </Caption>
     )),
     { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'containing an inline link and italicisation',
```

Comment:
```suggestion
    'containing italicisation',
```

This hasn't got an inline link such as [above](https://github.com/bbc/psammead/pull/446/files#diff-9a6daa50f38a2ed6254f1e71f42a1a43R38), probably  a copy paste mistake

Reply:
good spot! the link got refactored out, thank you 👍